spec/researches/passiveVoiceSpec.js

```diff
@@ -371,17 +367,212 @@ describe( "detecting passive voice in sentences", function() {
 
 	it( "returns no passive sentence when the subsentence has no auxiliary, when the auxiliary is used multiple times", function () {
 		// Passive: no passive, auxiliary: was
-		paper = new Paper( "He thought she was the one who knew about the six buried in his back yard, but he was wrong." );
+		var paper = new Paper( "He thought she was the one who knew about the six buried in his back yard, but he was wrong." );
 		expect( passiveVoice( paper ).passives.length ).toBe( 0 );
 	});
 
 	it( "returns no passive sentence when there is an exception with 'rid'", function () {
 		// Passive: no passive, auxiliary: got
-		paper = new Paper( "He got rid of it" );
+		var paper = new Paper( "He got rid of it" );
 		expect( passiveVoice( paper ) ).toEqual( {
 			total: 1,
 			passives: []
 		} );
 	});
 
 } );
+
+// German.
+describe( "detecting passive voice in sentences with irregularParticiples", function() {
```

Comment:
Putting the German tests in the same file feels wrong. If the code is extended with more languages we'll get excessive files with tests.